modules/ROOT/pages/install-rtfctl.adoc

```diff
@@ -556,6 +556,25 @@ Replicas:          1 desired | 1 updated | 1 total | 0 available | 1 unavailable
   hello-world-77f5897949-z54gb   Pending
 ----
 
+== Auditing rtfctl Operations
+
+The `rtfctl` command line tool (from vX.X.X and above) audits the `rtfctl` operations on the cluster. Helping cluster administrators and you to track operations that impact Runtime Fabric core software.
+
+Consider the following points:
+
+* Audit logs are stored in a Kubernetes secret in the `rtfctl-audit` namespace.
+* Audit logs are cluster scoped and not attached to Runtime Fabric core software lifecycle.
+* Command executions continue to function if the execution fails to audit the operation.
+
+To audit the operations, run the following command:
+----
+$ sudo ./rtfctl audit-logs
+<add a sample output>
+----
```

Comment:
We need a full example:

- The command to run
- An example of running the command
- The output example that returns

Reply:
command to run: 
`$ sudo ./rtfctl audit-logs`

output example that returns:
```
[{"timestamp":"2023-10-26 00:52:24.446 +0000 UTC","kubeContext":"rohit-rleasereadiness","command":["./rtfctl","restart","test-application","--app-namespace","8456f9c7-2149-4983-b02b-6b18ddd28efe","--namespace","rtf"]}]
```